microsoftgraph/microsoft-graph-explorer-v4
# Fix/toolkit component padding

## Overview
Solves #1444 

### Demo
![image](https://user-images.githubusercontent.com/45680252/154226862-6e551c75-ea8d-47fb-b476-5a7a1090a4a7.png)

## Testing Instructions
Click on 'my mail' in sample queries. 
Open toolkit tab on the response area
Resize screen to see how the text is displayed
